### DISCUSSION TITLE
Assistance needed in configuring Tractus-X EDC connectors with DIM (MIW)

### DISCUSSION THREAD

**Discussion by s-damith** [eclipse-tractusx/tractusx-edc#1605] [Q&A]
Hi,

I'm new to the dataspace topic and have been trying to configure two connectors(v8) using the latest DIM or MIW examples, but I've encountered some difficulties. I followed the [tutorial-resources](https://github.com/eclipse-tractusx/tutorial-resources) example, but it uses connector v6, while I need working examples for the latest version.

Specifically, I am struggling to provide the necessary configuration entries for my Tractus-X EDC Helm chart, such as:

**iatp.sts.dim.url**: the base URL for DIM.

Could anyone assist me in configuring the Decentralized Identity Management (DIM) system, or provide examples of working with the Tractus-X connector?

Additionally, can we use the EDC connector [IdentityHub](https://github.com/eclipse-edc/IdentityHub) for this task?

Any help would be greatly appreciated. Thank you!

**Comment by Moritz72** [comment]
I'm not directly involved in this project so take this with a grain of salt but this is my understanding of the current state of all of this.

With EDC version 0.6.x which roughly corresponds to TractusX-EDC version 0.7.x the DCP (formerly IATP) was adopted. Previously there was dedicated code in the TractusX-EDC that made it work together with the [MIW](https://github.com/eclipse-tractusx/managed-identity-wallet). This is also what the [MXD](https://github.com/eclipse-tractusx/tutorial-resources/tree/main/mxd) uses.

With the switch to EDC version 0.6.x the code for the MIW integration was removed. The MIW is currently being worked on to adopt DCP but there have not been any changes to the repository in the last two months. The endpoints are actually already there to make it work somewhat but there are still some issues that prevent it from working with the TractusX-EDC ([this](https://github.com/eclipse-tractusx/managed-identity-wallet/issues/347) is probably the most important one). [Here's](https://github.com/sap-contributions/eclipse-tractusx_tutorial-resources/tree/feat/mxd_0.8.0rc2) an attempt to update the MXD to use TracutsX-EDC version 0.8.x but I'm not sure if it's currently going anywhere.

There is a [wallet-stub](https://github.com/eclipse-tractusx/ssi-dim-wallet-stub) as well as the [umbrella charts](https://github.com/eclipse-tractusx/tractus-x-umbrella). With these you should be able to use newer Connector version. The DIM part is probably mocked for the umbrella charts but I have not tried any of these so I'm not sure.

Lastly there is the [MVD](https://github.com/eclipse-edc/MinimumViableDataspace). I would probably go with this one as it is currently maintained quite weill and also has quite some documentation. It does however not use TractusX-EDC but only the regular EDC in case that is something you car about. It is also currently missing proper credential issuance. But it uses the [Identity Hub](https://github.com/eclipse-edc/IdentityHub) so that would probably be a good place to learn how to use that as well.

**Comment by hemantxpatel** [comment]
To partially provide some context, MXD has been updated with traxtusx-edc `0.8.x` which uses indentity-hub. Identity HUB currently partially implements DIM protocol, so you might find some hackish solution in MXD to make it work. As of now there is no open source alternative for DCP.
